sdk/ml/azure-ai-ml/azure/ai/ml/operations/_azure_openai_deployment_operations.py

```diff
@@ -0,0 +1,46 @@
+# ---------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# ---------------------------------------------------------
+
+# pylint: disable=protected-access
+
+import logging
+from typing import Iterable
+
+from azure.ai.ml._restclient.v2024_04_01_preview import AzureMachineLearningWorkspaces as ServiceClient2020404Preview
+from azure.ai.ml._scope_dependent_operations import OperationConfig, OperationScope, _ScopeDependentOperations
+from azure.ai.ml.entities._autogen_entities.models import AzureOpenAIDeployment
+
+from ._connections_operations import ConnectionsOperations
+
+module_logger = logging.getLogger(__name__)
+
+
+class AzureOpenAIDeploymentOperations(_ScopeDependentOperations):
+    def __init__(
+        self,
+        operation_scope: OperationScope,
+        operation_config: OperationConfig,
+        service_client: ServiceClient2020404Preview,
+        connections_operations: ConnectionsOperations,
+    ):
+        super().__init__(operation_scope, operation_config)
+        self._service_client = service_client.connection
+        self._connections_operations = connections_operations
+
+    def list(self, connection_name: str, **kwargs) -> Iterable[AzureOpenAIDeployment]:
```

Comment:
Add simple docstring.

Reply:
I am planning to add docstrings for serverless and AOAI in a follow-up PR